contribs/gmf/src/services/themesservice.js

```diff
@@ -51,6 +56,12 @@ gmf.Themes = function($http, treeUrl, isThemePrivateUrl) {
   this.$http_ = $http;
 
   /**
+   * @type {angular.$q}
+   * @private
+   */
+  this.q_ = $q;
```

Comment:
Please, let me know if I'm mistaken.  See:

https://google.github.io/styleguide/angularjs-google-style.html

Look under "2 Angular Style Rules", Compare the "yes" and "no" there.

>Do not use $ to prepend your own object properties and service identifiers. Consider this style of naming reserved by AngularJS and jQuery.

>Why? It's useful to distinguish between Angular / jQuery builtins and things you add yourself. In addition, $ is not an acceptable character for variables names in the JS style guide.

Do I understand correctly that we shouldn't call our inner variables with a `$` for angular objects ?